resources/ory/charts/hydra/values.yaml

```diff
@@ -16,6 +18,10 @@ nameOverride: ""
 # Full chart name override
 fullnameOverride: ""
 
+## Pod priority
+## https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+# priorityClassName: ""
+
```

Comment:
Why this is commented out?

Reply:
It's copied from ORY charts.

Reply:
It was a copy from ORY charts, I've removed it.